js/src/components/google-combo-account-card/account-details.js

```diff
@@ -17,13 +17,14 @@ import useGoogleMCAccount from '.~/hooks/useGoogleMCAccount';
 const AccountDetails = () => {
 	const { google } = useGoogleAccount();
 	const { googleAdsAccount } = useGoogleAdsAccount();
-	const { googleMCAccount } = useGoogleMCAccount();
+	const { googleMCAccount, isReady: isGoogleMCConnected } =
```

Comment:
Nitpick: destructuring `isReady: isGoogleMCConnected` is a bit confusing since this hook also returns a `hasGoogleMCConnection` prop. 

Suggest:
```suggestion
	const { googleMCAccount, isReady: isGoogleMCReady } =
```

Reply:
👍 . Updated